uk.co.compucorp.civicrm.hrcomments/info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.hrcomments" type="module">
+  <file>hrcomments</file>
+  <name>Comments</name>
+  <description>Track user comments</description>
+  <license>AGPL-3.0</license>
+  <maintainer>
+    <author>Compucorp Ltd</author>
+    <email>info@compucorp.co.uk</email>
+  </maintainer>
+  <releaseDate>2017-01-12</releaseDate>
+  <version>1.0</version>
+  <develStage>alpha</develStage>
+  <compatibility>
+    <ver>4.2</ver>
+  </compatibility>
+  <comments>This module enables commenting functionality</comments>
```

Comment:
Since this will be almost the same as the description, it's better to leave it empty